src/python/pants/engine/target.py

```diff
@@ -100,11 +101,14 @@ def compute_value(cls, raw_value: Optional[int], *, address: Address) -> Optiona
                 return value_or_default
     """
 
-    # Subclasses must define this.
+    # Subclasses must define these.
     alias: ClassVar[str]
+    help: ClassVar[str]
```

Comment:
Some explanation of how each of these should be shaped ("supports newlines") and how they are consumed would be good.